spec/lib/sequent/core/helpers/type_conversion_support_spec.rb

```diff
@@ -264,6 +264,47 @@ class CommandWithDateTime < Sequent::Core::BaseCommand
 
   end
 
+  context Hash do
+    class CommandWithHash < Sequent::Core::BaseCommand
+      attrs value: Hash
```

Comment:
I don't think a Hash containing a ValueObject works. Can you add a test for this? I think if it does not work we should fail fast, and document that it is not supported.

Reply:
Yes I was going to fix this.